scripts/build.sh

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env bash
 
-if [ -z "$1" ]; then 
+if [ -z "$1" ]; then
     BUILD_TYPE=Release
-else 
+else
     BUILD_TYPE="$1"
 fi
 
-cd $(dirname "$0")
-mkdir -p build
-cd build
-cmake .. -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
+cd $(dirname "$0")/.. &&
```

Comment:
That is nice too, however a bit different. That hardcodes the script must be located in the script directory, while dirname hardcodes the script must be one level up from root.